controls/pcidss_4.yml

```diff
@@ -2085,18 +2085,28 @@ controls:
         - cracklib_accounts_password_pam_ucredit
 
     - id: 8.3.7
-      title: 'Individuals are not allowed to submit a new password/passphrase that is
-        the same as any of the last four passwords/passphrases used.A previously used
-        password cannot be used to gain access to an account for at least 12 months.This
-        requirement is not intended to apply to user accounts on point-of-sale terminals
-        that have access to only one card number at a time to facilitate a single
-        transaction (such as IDs used by cashiers on point-of-sale terminals).'
+      title: Individuals are not allowed to submit a new password/passphrase that is the same as
+        any of the last four passwords/passphrases used.
+      description: |-
+        A previously used password cannot be used to gain access to an account for at least 12
+        months.
       levels:
         - base
       status: automated
+      notes: |-
+        This requirement is not intended to apply to user accounts on point-of-sale terminals that
+        have access to only one card number at a time to facilitate a single transaction (such as
+        IDs used by cashiers on point-of-sale terminals).
+        For RHEL 8 and RHEL 8 systems, the accounts_password_pam_pwhistory_... rules should be
```

Comment:
Should this be RHEL 8 and RHEL 9?

Reply:
Fixed in 5f6665a860ca87ae23caf173d48189366949121c . Thanks